core/deployment/src/main/java/io/quarkus/deployment/builditem/GraalVmFeatureBuildItem.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.deployment.builditem;
+
+import java.util.Objects;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Represents a GraalVM feature to be passed to native-image through the {@code --features} options.
+ */
+public final class GraalVmFeatureBuildItem extends MultiBuildItem {
```

Comment:
```suggestion
public final class GraalVMFeatureBuildItem extends MultiBuildItem {
```

Reply:
Good catch. I changed all `[Gg]raalVm` prefixes to `[Gg]raalVM`

Reply:
In general all our native image build items are prefixed with `NativeImage` rather than `GraalVM`.

Reply:
Fixed